### PR TITLE
[master < add-mage-1.5-changelog] Add mage 1.5 changelog

### PR DESCRIPTION
**Pull request by antoniofilipovic**
### Description

Add changelog for MAGE 1.5

### Pull request type

Please delete options that are not relevant and check the ones that are.

- [x] Documentation improvements

### Checklist:

- [x] I checked all content with Grammarly
- [x] I performed a self-review of my code
- [x] I made corresponding changes to the rest of the documentation
- [x] The build passes locally
- [x] My changes generate no new warnings or errors

### CODE REVIEW

File: mage/changelog.md [hunk 1]
```diff
@@ -4,6 +4,21 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## v1.5 - Dec 20, 2022
+
+### Major Features and Improvements
+
+- Added support for ancestors, descendants, topological sort, connecting nodes, and chaining nodes.
+  [#180](https://github.com/memgraph/mage/pull/180)
+- Aligned C++ API with Memgraph 2.5
```
Comment (vpavicic) [inline mage/changelog.md:13]:
```suggestion
- C++ API is now aligned with Memgraph 2.5
```

File: mage/changelog.md [hunk 2]
```diff
@@ -4,6 +4,21 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## v1.5 - Dec 20, 2022
+
+### Major Features and Improvements
+
+- Added support for ancestors, descendants, topological sort, connecting nodes, and chaining nodes.
```
Comment (vpavicic) [inline mage/changelog.md:11]: now you can do this this this and this how?

File: mage/changelog.md [hunk 3]
```diff
@@ -4,6 +4,21 @@ title: Changelog
...
+  [#180](https://github.com/memgraph/mage/pull/180)
+- Aligned C++ API with Memgraph 2.5
+  [#184](https://github.com/memgraph/mage/pull/184)
+- Made Graph Coloring output vertices and integers instead of strings
```
Comment (vpavicic) [inline mage/changelog.md:15]:
```suggestion
- Graph Coloring no longer outputs strings but vertices and integers.
```
Why is this important? what can i do with vertices and integers i couldn't do with strings?

File: mage/changelog.md [hunk 4]
```diff
@@ -4,6 +4,21 @@ title: Changelog
...
+  [#177](https://github.com/memgraph/mage/pull/177)
+
+### Bug Fixes
+- Enable module reset to be able to train and evaluate without shutting down database, enable working with class labels which don't start from 0, and fix potential early stopping due to low limit in the Node classification module.
+  [#173](https://github.com/memgraph/mage/pull/173)
```
Comment (vpavicic) [inline mage/changelog.md:20]: By DOING THIS you can now train and evaluate WHAT without shutting down the database
Class labels can now start from 0
Low limit of WHAT no longer prematurely stops WHAT while running the Node classification module

**Comment by vpavicic** [comment]
As we no longer have blog posts for minor releases, we need to make changelog as user friendly as possible
so imagine this isn't just a changelog anymore .. this is also the blogpost/newsletter you want to tell the user how their life is better now :D

File: mage/changelog.md (hunk 2 above)
Comment (antoniofilipovic) [reply mage/changelog.md:11]: Fixed

File: mage/changelog.md (hunk 3 above)
Comment (antoniofilipovic) [reply mage/changelog.md:15]: You can use them further in Lab and so on.

**Comment by antoniofilipovic** [comment]
> As we no longer have blog posts for minor releases, we need to make changelog as user friendly as possible so imagine this isn't just a changelog anymore .. this is also the blogpost/newsletter you want to tell the user how their life is better now :D

@vpavicic fixed this now. Makes sense what you said.

File: mage/changelog.md [hunk 5]
```diff
@@ -4,6 +4,23 @@ title: Changelog
...
+- C++ API is now aligned with Memgraph 2.5
+  [#184](https://github.com/memgraph/mage/pull/184)
+- Graph Coloring no longer outputs strings but vertices and integers. Now you can use result of graph coloring directly in Memgraph Lab for example.
+  [#177](https://github.com/memgraph/mage/pull/177)
+
+### Bug Fixes
+- By enabling module reset you can now train and evaluate model without shutting down the database.
+  Class labels can now start from 0 or negative numbers.
+  Low limit of early stopping flag no longer prematurely stops training of model while running the Node classification module.
+  [#173](https://github.com/memgraph/mage/pull/173)
```
Comment (vpavicic) [inline mage/changelog.md:22]:
```suggestion
- Now you can find ancestors (all the nodes from which a path exists ) and descendants (all nodes to which a path exists) starting from a certain node, sort directed acyclic graph in a way that a node which appears before others is first, return a subgraph from nodes using `connect_nodes` method, and create relationships between nodes in a list using the `chain_nodes` method.
  [#180](https://github.com/memgraph/mage/pull/180)
- C++ API is now aligned with Memgraph 2.5
  [#184](https://github.com/memgraph/mage/pull/184)
- Graph Coloring no longer outputs strings but vertices and integers. This allows you to use the result of graph coloring directly in Memgraph Lab.
  [#177](https://github.com/memgraph/mage/pull/177)

### Bug Fixes
- By enabling module reset, you can now train and evaluate the model without shutting down the database.
  Class labels can now start from 0 or negative numbers.
  The low limit of the early stopping flag no longer prematurely stops the training of the model while running the Node classification module.
  [#173](https://github.com/memgraph/mage/pull/173)
```

**Pull request merged**